showcase/src/app/demo/typography/demo-typography.component.html

```diff
@@ -0,0 +1,54 @@
+<h1 class="mat-display-2" translate>SHOWCASE.DEMO.TYPOGRAPHY.TITLE</h1>
+<section class="stark-section">
+	<h1 translate>SHOWCASE.DEMO.TYPOGRAPHY.MATERIAL_BASED.TITLE</h1>
+	<p [innerHTML]="'SHOWCASE.DEMO.TYPOGRAPHY.MATERIAL_BASED.DETAILS' | translate"></p>
+</section>
+<section class="stark-section">
+	<h1 translate>SHOWCASE.DEMO.TYPOGRAPHY.FONT_FAMILY.TITLE</h1>
+	<p translate>SHOWCASE.DEMO.TYPOGRAPHY.FONT_FAMILY.DETAILS_1</p>
+	<p translate>SHOWCASE.DEMO.TYPOGRAPHY.FONT_FAMILY.DETAILS_2</p>
+	<p>
+		<stark-pretty-print [data]="fontFamilyCSS" format="css" [enableHighlighting]="true"></stark-pretty-print>
+	</p>
+</section>
+
+<section class="stark-section">
+	<h1 translate>SHOWCASE.DEMO.TYPOGRAPHY.CLASSES_AND_TAGS.TITLE</h1>
+	<example-viewer class="mb3" [extensions]="['HTML']" filesPath="typography/stark" exampleTitle="STARK classes">
+			<h1>Headline 1</h1>
+			<h2>Headline 2</h2>
+			<h3>Headline 3</h3>
+			<h4>Headline 4</h4>
+	</example-viewer>
+	<example-viewer [extensions]="['HTML']" filesPath="typography/mat" exampleTitle="Angular Material classes">
+			<div class="mat-display-4">Display 4</div>
```

Comment:
Could you maybe change the text from "Display 4" to "Mat-Display-4" so it is clear what is the class being used?

The same for the others in this section